resource/base/segments_traits_operators.md

```diff
@@ -59,6 +65,29 @@ Traits use the following syntax: `<segment>:<trait>`
 
 | **Trait** | **Description** | **Valid Types** |
 | `base:universal_access` | Set a resource's **Universal Access** level. | `str` |
+| `cdtinfo:case_allowed` | Set the **Case Allowed** for the names of profiles in this resource class. | `str`, `False` |
+| `cdtinfo:default_racroute_return_code` | Set the **Default Return Code** for requests to access profiles in this resource class. | `int`, `False` |
+| `cdtinfo:valid_first_characters` | Set the **Valid Character Types** for the first characters in names of profiles in this resource class. | `str`, `False` |
+| `cdtinfo:grouping_class_name` | Set the **Grouping Class** name associated with this resource class. | `str`, `False` |
+| `cdtinfo:key_qualifiers` | Set the number of **Qualifiers** used to determine a match for profiles in this resource class | `int` |
+| `cdtinfo:max_length` | Set the **Maximum Length** for the names of profiles within this resource class. | `str`, `False` |
+| `cdtinfo:max_length_entityx` | Set the **Maximum Length** for the names of profiles when a RACROUTE macro is invoked with the ENTITYX keyword within this resource class. | `str`, `False` |
+| `cdtinfo:valid_other_characters` | Set the **Valid Character Types** for the other characters in names of profiles in this resource class. | `str`, `False` |
+| `cdtinfo:posit_number` | Set the **Posit Number** for this resource class. This controls flags that control many RACF processing options | `int`, `False` |
```

Comment:
Incomplete Description?

Reply:
Missing '.' at the end of one description. Now fixed.